### PR TITLE
chore: remove SendMsgDeliveryGroup Cmd

### PR DESCRIPTION
**Pull request by joshuef**
This is not needed or really utilised in any real way.

We can just use SendMsg cmds

### CODE REVIEW

File: sn_node/src/node/api.rs [hunk 1]
```diff
@@ -164,9 +164,8 @@ impl Node {
         let dst_pk = self.section_key_by_name(&target_name);
         wire_msg.set_dst_section_pk(dst_pk);
 
-        let cmd = Cmd::SendMsgDeliveryGroup {
+        let cmd = Cmd::SendMsg {
             recipients: targets.into_iter().collect(),
-            delivery_group_size: dg_size,
```
Comment (oetyng) [inline sn_node/src/node/api.rs:169]: Seems logging on line 160 will be confusing with the dg_size mentioned.

Comment (joshuef) [reply sn_node/src/node/api.rs:169]: ah yeh, i wonder if i can just strip that whole dg stuff out 🤔

Comment (joshuef) [reply sn_node/src/node/api.rs:169]: hmm, maybe not in this pass. i'll clean that log up 👍

**Review by oetyng — APPROVED** [review]

**Pull request merged**